.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+<!-- Following this template is optional. If you don't feel like it applies to your change feel free to delete it. !-->
+
+## Describe the Problem and Solution
+<!-- Write 2-3 sentences summarising the reasons behind this change (this might be the problem you're solving, or the context around the request) and the solution you have chosen. !-->  
```

Comment:
Given we are moving to the bot posted comment methodology. I think we can convert the template so that it is simply this one line:
```Summarize the reason behind this change (this might be the problem you're solving, or the context around the request) and the solution you have chosen.```
No markdown or anything. This matches what MongoDB does too. Then we take the remainder of the template and move it to the bot posted comment. From "What makes this change safe?" onward should be in the posted comment, including that heading.

Reply:
Updated as suggested